config.go

```diff
@@ -60,6 +63,18 @@ func (cfg Config) MakeLabels(types []string) string {
 	return labels
 }
 
+func (cfg Config) SavePropertyNativeType(node graph.Node, expandedPropertyKey, val string) interface{} {
```

Comment:
GetNativeProperyValue
Add comments for all exported funcs
Say: this is the func to call before you save stuff... etc.